CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Features
 - Show assigned facility in the patient summary screen
 - Use patient's assigned facility to schedule an appointment
+- Show `Transferred to` label in Overdue screen
```

Comment:
Can we add an additional changelog item that indicates that overdue was changed to load patients whose assigned facility is the current facility?

Reply:
Done, added it to `Changes` section